<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>密钥管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>对称密钥</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox">
             <property name="value">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>生成</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>密钥的长度</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>对称密钥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>非对称密钥</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>0</height>
              </size>
             </property>
             <property name="title">
              <string>操作</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>添加</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>生成密钥对</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_2"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>加密的文件路径</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>生成</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>签名</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="2" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>文件路径</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="4" column="1" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>签名</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>选择路径</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0" rowspan="3">
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>账户管理</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>操作</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>密 码：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>用户名：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>安全性：</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>清空</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_3">
           <property name="showGrid">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
